cogs/utils/github.py

```diff
@@ -13,9 +12,21 @@ def check_for_updates(version):
     return False
 
 def changelog():
-    latest = get_latest()
-    response = requests.get(f"https://github.com/r-liner/discord-commands-bot/releases/tag/v{latest}")
-    soup = BeautifulSoup(response.text, "html.parser")
-    _changelog = soup.find("div", class_="markdown-body my-3").get_text()
+    # Use text media_type: https://docs.github.com/en/rest/using-the-rest-api/getting-started-with-the-rest-api#media-types
```

Comment:
Remove comment

Reply:
That's reference to official GitHub api not sure why comment should be removed, but okay